src/mailing/sender.js

```diff
@@ -1,6 +1,8 @@
 const nodemailer = require('nodemailer');
 const Email = require('email-templates');
 const path = require('path');
+const ejs = require('ejs');
```

Comment:
If you switch to using `email-templates` then this will no longer be needed

Reply:
Removed